dynamic_vino_lib/src/models/object_detection_ssd_model.cpp

```diff
@@ -104,9 +105,9 @@ bool Models::ObjectDetectionSSDModel::fetchResults(
   }
 
   slog::debug << "Fetching Detection Results ..." << slog::endl;
-  InferenceEngine::InferRequest::Ptr request = engine->getRequest();
+  ov::InferRequest request = engine->getRequest();
   std::string output = getOutputName();
-  const float * detections = request->GetBlob(output)->buffer().as<float *>();
+  float * detections = (float * )request.get_tensor(output).data();
```

Comment:
Hi,  Why there remove the const flag.  according the code rule and code scan it should keep the const, if the point value no the change.